SQF/dayz_code/compile/A_Plot_for_Life/fn_check_owner_friends.sqf

```diff
@@ -49,7 +49,7 @@ if (DZE_permanentPlot) then {
 	_owner = (_targetOwner == dayz_characterID);
 };
 
-if (_targetOwner in _friendlies) then {
+if (_player in _friendlies) then {
```

Comment:
Sorry, this was my fault. You are absolutely correct about _player and _friendlies. But searching the _target owner's ID (=_targetOwner) in the _friendlies list of the _target always returns true, when DZE_permanentPlot is true. Instead it should search for the _playerUID in _friendlies.

Please correct me if I am totally wrong.